### DISCUSSION TITLE
Offer. Адаптация плагина HideTag

### DISCUSSION THREAD

**Discussion by azarro** [madromas/esoTalk-v3#4] [Q&A] [answered]
Возможна адаптация плагина  ( **_HideTag_** Enable [hide], [hide=100], [groups=1,4], [users=Toby,Tristan] and [visitor] tags to hide content.)

**Comment by madromas** [comment]
Где вы это нарыли? В архиве esotalk.org? И что там про это написано?

**Reply by azarro** [reply]
В старой версии esoTalk он был, и прекрасно работал. До сих пор его использую на сайте. 
Посмотреть ( https://workupload.com/file/xbAwRCZpTfM )
[hide=100] - показывать скрытый контент после 100 сообщений 
[groups=1,4] - показывать скрыты контент только тем кто находиться в группах 1 и 4
[users=Toby,Tristan] - показывать скрытый контент только перечисленным юзерам

**Reply by madromas** [reply]
Дак он же есть в этой сборке. Или это не то? 

Если он работает у вас, то в чем проблема? Вы хотите чтоб я его в плагины поместил? Или он на новой версии не пашет?

Чего то я не понял.

**Reply by azarro** [reply]
В вашей сборки его нет. Было бы не плохо, если бы он вошел в вашу сборку. Его ведь надо адаптировать к версии PHP 8.

**Reply by madromas** [reply]
Дак есть же плагин хиде. Ладно, сеичас гляну что там у вас за плагин.

**Reply by madromas** [reply]
Может у вас еше какие плагины есть. Выкладывайте. Будем смотреть. Может чего и адаптируем.

**Reply by azarro** [reply]
У вас в папке Addons/Plugins не вижу такого плагина. Может он как-то по другому называется и имеет другие настройки. Подскажите название.

**Reply by madromas** [reply]
SpoilerTag

**Reply by azarro** [reply]
Да вроде все, остальные такие-же как у вас но меньше по количеству. Кстати плагин  **sitemap** не работает, на старой версии то-же не работал.

**Reply by azarro** [reply]
Если мне не изменяет память, то этот плагин SpoilerTag - добавляет кнопку Спойлер (Т.е скрыть портянку текста по спойлер) в редакторе.

**Reply by madromas** [reply]
Как так? У меня работает https://hub.madway.net/sitemap.xml. Я же говорил, вам надо всё удалить и залить по новой. 

Плагин Хайд, действительно другой, буду смотреть.

**Reply by azarro** [reply]
Не знаю в чем проблема, я все снес и базу тоже. Скачал у вас с ГитХаба последние файлы, установил. Проблема не исчезла.

**Reply by madromas** [reply]
Не знаю как так, но в вашем сайте при регистрации, есть ошибка. Включите дебуг в конфигурации и попробуйте заригить нового юзера.

**Comment by madromas** [comment]
Вы в настройки плагина Sitemap заходили? 

ПС. Ваш плугин Хайд, работает на php8. Только надо его в языковый пакет занести, и поправить CSS.

https://hub.madway.net/22-topic-for-testing/p1#p211

**Reply by azarro** [reply]
Включил плагин Sitemap, сайт опять упал в ошибки. Опять картинка - что-то пошло не так... Буду пробовать на XAMP'e.

**Reply by madromas** [reply]
Я действительно не знаю что там у вас происходит. Сам попробую на ХАМПП поставить, может я какие файлы забыл обновить

**Reply by madromas** [reply]
Сейчас, весь архив обновил. Надеюсь всё поправлено

**Reply by azarro** [reply, marked as answer]
Обновил файлы, и всё ЗАРАБОТАЛО

**Reply by azarro** [reply]
**В подвале ошибка**
Warning: Undefined array key "metaMenuItems" in /home/admin/web/esotalk3.anclav24.keenetic.link/public_html/core/views/default.master.php on line 141

**Reply by madromas** [reply]
Когда всё работает, то Debug надо ставить False

**Reply by azarro** [reply]
Все ошибка исчезла, после включения плагина **FooterLinks** Adds links to footer.

**Reply by madromas** [reply]
Обновите языки и Дарк скин. Я подправил Хайд плагин.

**Reply by madromas** [reply]
Перезалейте плагин Хайд тоже
